src/app/organizations/state/register-organization.service.ts

```diff
@@ -49,6 +50,12 @@ export class RegisterOrganizationService {
   // The old regex in case needed
   // readonly urlRegex = /https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,63}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)/;
   readonly organizationNameRegex = /^[a-zA-Z][a-zA-Z\d]*$/;
+
+  readonly logoUrlRegex = new RegExp(
```

Comment:
Need a unit test for this regex, see src/app/organizations/state/register-organization.service.spec.ts for similar test.

Missing a $ at the end `.gif)$` otherwise `.png potato` works.  Also check that `.png` does not work

Reply:
Added tests, and slightly changed regex so things like `.png potato` or`.png` will fail.